interface/modules/zend_modules/module/Patientvalidation/src/Patientvalidation/Controller/BaseController.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace Patientvalidation\Controller;
+
+use Zend\InputFilter\InputFilter;
+use Zend\Mvc\Controller\AbstractActionController;
+use Application\Listener\Listener;
+use Zend\Mvc\Controller\ActionController;
+use Zend\View\Model\ViewModel;
+
+class BaseController extends AbstractActionController
+{
+    private $configParams = null;
+
+
+
+    public function __construct()
+    {
+        //load translation class
+        $this->translate = new Listener();
+    }
+
+
+    protected function getLanguage(){
+
+        $sm = $this->getServiceLocator();
+        $dbAdapter = $sm->get('Zend\Db\Adapter\Adapter');
+        $sql = new CustomSql($dbAdapter);
+
+        $lang = $sql->getCurrentLang();
+
+        return $lang;
+    }
```

Comment:
You can use this function to determine what language the openemr system runs on and this way to use RTL or LRT in your view and so on.